documentation/guides/general_setup.md

```diff
@@ -0,0 +1,230 @@
+# General setup guide
+
+This is an exhaustive list for new developers. Feel free to skip the steps that
+you have already done on your own.
+
+## Operating system
+
+```{note}
+Most of this is also true for many other OSS projects.
+```
+
+```{caution}
+While we would like to, Openverse currently does not support development
+natively on Windows because we have no active contributors who use Windows on a
+daily basis. Nonetheless, PRs that enable or improve the Openverse development
+experience on Windows are welcome.
+```
+
+```{tip}
+We recommend Windows Subsystem for Linux (WSL) as a viable option.
+```
```

Comment:
This all looks SO good when rendered ✨

Reply:
Yes, it's awesome. Furo is a good theme.